crates/storage/src/config.rs

```diff
@@ -0,0 +1,2 @@
+pub const REDB_FILE: &str = "ream.redb";
+pub const REDB_CACHE_SIZE: usize = 1_024;
```

Comment:
what unit is this const in? could you add a doc comment. Is this supposed to be mb, gb, kb etc